README.md

```diff
@@ -11,20 +77,24 @@
 - [ ] Runs in the same security sandbox as JavaScript frameworks like Angular, React or Vue
```

Comment:
We have Blazor Server now at frontend side, we want to update it? Or we will later make a transition to Blazor WebAssembly, so there will be no point in updating this chapter?